torch/jit/_monkeytype_config.py

```diff
@@ -0,0 +1,115 @@
+# !/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
This file will not be run independently, likely we won't need these two lines.

Reply:
I agree, but flake8 complains. Hence, I added this.

Reply:
Looks like it doesn't complain. I will remove this